docs/blog/newsletter-july-2023.rst

```diff
@@ -0,0 +1,79 @@
+
+Often, that discussion is enough to resolve the problem. But if your manager isn't responsive, it becomes a matter of holding to your boundaries. In this case, consider providing your manager with a breakdown that describes each task, how long it will take, and why. You can point to this breakdown to answer questions about why tasks aren't getting done. Task boards like Jira and Trello can help with this and also making others aware of the problem because reporters can be notified when their tickets are affected by new requests.
+
+Another idea is taking a few days of vacation so that others have to fill in for you and learn by experience how overloaded you are. Or when you get an impossible request, respond by describing an alternative you *can* deliver instead of explaining why you can't achieve the unachievable. If possible, take unclear requests off of your schedule until you understand the business need and requirements. A last resort is taking the problem to an intermediary like a colleague, another manager, or Human Resources.
+
+When all else fails, you can continue working on tasks at a slow enough pace to prevent burnout, prioritizing as you go and doing your best to ignore unfair criticism and attempts to make you feel guilty. If important tasks are delayed because you're overloaded, it might eventually draw enough attention to the problem to get it solved. If you're worried about getting fired for taking this approach, consider spending the time you save on looking for a new job or finding some freelance work to mitigate the risk and help you get comfortable with moving on.
+
+------------------------
+Getting Info from Others
+------------------------
+
+A recent thread resurrected an enduring issue with documentarians: How do we get the information we need from experts? The `May 2020 newsletter </blog/newsletter-may-2020/#getting-more-info-from-smes>`__ addressed this issue. The current discussion focused on how to ask questions of developers: what questions to ask and how to get the best responses for documentation. 
+
+People indicated that there are two levels of questions: 1) general, high-level questions; and 2) specific, detailed questions. Documentarians ask general questions to get an overall understanding of a product or domain. After we have the higher-level understanding, we can determine what detailed questions to ask. Specific questions fill in the gaps in our knowledge.
+
+For general questions, you can ask them of product managers or customer support. Another potential resource could be any QA testers. We also have `learning resources on the topic </topics/#working-with-other-roles>`__ for interacting with people in other positions. If you're writing end-user documentation, this knowledge along with your personal experience of the product may be sufficient.
```

Comment:
In the first sentence, maybe "For general questions, you can ask product managers..." (remove the `them of`)?

Reply:
Sounds better to me.

```suggestion
For general questions, you can ask product managers or customer support. Another potential resource could be any QA testers. We also have `learning resources on the topic </topics/#working-with-other-roles>`__ for interacting with people in other positions. If you're writing end-user documentation, this knowledge along with your personal experience of the product may be sufficient.
```